.appveyor.yml

```diff
@@ -93,7 +93,7 @@ for:
       - sh: "source ${HOME}/venv${PYTHON_VERSION}/bin/activate"
       - sh: "rvm use 2.5"
       - sh: "nvm install ${nodejs_version}"
-      - sh: "npm install npm@5.6.0 -g"
+      - sh: "npm install npm@5.7.0 -g"
```

Comment:
seems like the previous version doesn't support `npm ci`. Do I just update to the closest version that does (5.7)? Or should I update to the latest veresion?

Reply:
I recommend bumping the npm version to one that supports Node14, since you'll also need to bump the Node version in Appveyor as Node10 is deprecated. You can refer to #307 for these version bumps and the lock file cleanup.